.NET/Microsoft.Recognizers.Text.Number/Parsers/BaseCJKNumberParser.cs

```diff
@@ -132,7 +132,7 @@ protected ParseResult ParseFraction(ExtractResult extResult)
             }
             else
             {
-                intPart = "零";
+                intPart = Config.ZeroChar.ToString();
```

Comment:
If you need to convert to String, why not declare as string in the YAML or in the Config object?
Otherwise you're doing unnecessary conversions all the time.

Reply:
Yes, i need to convert to String here, because "零" should be a String here, but in the latter when we  judge " intStr[i] == '零' "，零 should be a char